main.go

```diff
@@ -81,8 +90,9 @@ BE ON THE LOOKOUT !
 		select {
 		case input := <-inputChannel:
 
-			if input != "" {
-				fmt.Println("Snap")
+			if input == "" {
```

Comment:
This shouldn't be needed? Any time there is something in the input channel we want to snap, I think?